### ISSUE TITLE
Video BGA not affected by Rate Modifier

### ISSUE THREAD

**Issue by marysiamzawka** [seraxis/lr2oraja-endlessdream#33]
When using Rate Modifier on a song with a video BGA, the playback speed of that BGA is not affected. May not be an issue at all.

**Comment by seraxis** [comment]
Good catch! BGA isn't loaded in practice mode so I completely forgot about this
